### DISCUSSION TITLE
Ask for age in NSFW Channels only once per channel

### DISCUSSION THREAD

**Discussion by Andre19926** [stoatchat/discussions#534] [Feature Suggestions]
Greetings,
I got some feedback from people who are on my server and they agree that having to verify that you are 18+ every single time you re-enter a channel can get very annoying.

It would be nice of you only had to do that once per channel instead of every time each visit.

Cheers

**Comment by nulldg** [comment]
+1
its current implementation is abysmal

**Comment by Camwoodstock** [comment]
+1, the fact it doesn't actually save this seemingly almost 2 years after this initial suggestion is... Extremely dubious.
